src/coreclr/jit/compiler.h

```diff
@@ -79,6 +79,8 @@ class FgStack;             // defined in fgbasic.cpp
 class Instrumentor;        // defined in fgprofile.cpp
 class SpanningTreeVisitor; // defined in fgprofile.cpp
 class CSE_DataFlow;        // defined in OptCSE.cpp
+class OptBoolsDsc;         // defined in optimizer.cpp
+struct OptTestInfo;
```

Comment:
You probably don't need this second forward declaration.
```suggestion
```

Reply:
Done.